services/community-site/src/pages/Staking/Delegation.tsx

```diff
@@ -121,39 +121,38 @@ const Delegation = ({ location }: { location: Location }) => {
       (async () => {
         setLoadingAccountData(true);
         setDelegations(await getDelegations(account));
+        setUndelegations(await getUndelegations(account));
         setLoadingAccountData(false);
```

Comment:
What happens if there's an error with one of the two statements above? I think it will load continuously

Reply:
Yeah, correct. Better to keep it separate as it was.